NitroxClient/MonoBehaviours/PlayerMovementBroadcaster.cs

```diff
@@ -11,7 +11,7 @@ namespace NitroxClient.MonoBehaviours;
 
 public class PlayerMovementBroadcaster : MonoBehaviour
 {
-    public const float BROADCAST_INTERVAL = 0.05f;
+    public const float BROADCAST_INTERVAL = 0.12f;
```

Comment:
Maybe we should put a lil comment there to explain that we shouldn't go down this value because of weird behaviours of the physic engine

Reply:
We might need to change how movement is handled by putting it in a [FixedUpdate](https://docs.unity3d.com/ScriptReference/MonoBehaviour.FixedUpdate.html). So when a movement update is received, wait until physics frame and set velocity then. Might solve this weird behavior. Or at least something to look into in the future by adding a TODO comment here.